gulpfile.js

```diff
@@ -217,6 +217,23 @@ gulp.task('node-inspector', function () {
     }));
 });
 
+// wiredep task
+gulp.task('wiredep', function () {
+  return gulp.src('config/assets/default.js')
+    .pipe(plugins.wiredep({
+      ignorePath: '../../',
+      fileTypes: {
+        html: {
+          replace: {
```

Comment:
I don't understand what this is doing. Can you elaborate on your explanation? I'm curious why the *script* & *link* directives come into play here.